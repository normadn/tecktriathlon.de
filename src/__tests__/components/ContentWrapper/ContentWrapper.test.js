import React from "react";
import ContentWrapper from "../../../components/ContentWrapper";
import { shallow } from "enzyme";

describe("ContentWrapper", () => {
  it("renders correctly", () => {
    expect(shallow(<ContentWrapper />)).toMatchSnapshot();
  });
});
