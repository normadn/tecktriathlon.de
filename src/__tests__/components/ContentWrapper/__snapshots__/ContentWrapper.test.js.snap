// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper renders correctly 1`] = `
<div
  css={
    Object {
      "@media screen and (max-width: 1200px) and (min-width: 860px)": Object {
        "paddingLeft": "80px",
        "paddingRight": "80px",
      },
      "@media screen and (max-width: 860px)": Object {
        "paddingLeft": "5%",
        "paddingRight": "5%",
      },
      "height": "100%",
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": "1400px",
      "paddingLeft": "145px",
      "paddingRight": "145px",
    }
  }
/>
`;
